src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/StandardOleMarshalObject.Windows.cs

```diff
@@ -21,6 +21,7 @@ protected StandardOleMarshalObject()
 
         private IntPtr GetStdMarshaler(ref Guid riid, int dwDestContext, int mshlflags)
         {
+            Debug.Assert(OperatingSystem.IsWindows());
```

Comment:
Should the whole `StandardOleMarshalObject` class be marked as `SupportedOSPlatformAttribute("windows")`? Just like COM, I think `Ole` is Windows-specific technology.

Reply:
I am not sure, the file has Windows prefix `StandardOleMarshalObject.Windows.cs` and there is a unix version `StandardOleMarshalObject.Unix.cs`, though unix version has no any implementation https://github.com/dotnet/runtime/blob/01e82df18426d2924bc1b6c53cf6de1b7752eadb/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/StandardOleMarshalObject.Unix.cs